src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SimplifyBooleanExpressionCheck.java

```diff
@@ -39,6 +39,31 @@
  * <pre>
  * &lt;module name=&quot;SimplifyBooleanExpression&quot;/&gt;
  * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * public class Test {
+ *
+ *  public void bar(){
+ *
+ *   boolean a, b;
+ *   Foo c, d, e;
+ *
+ *   if(!false){}; // violation, can be simplified to true
+ *
+ *   if(a == true){}; // violation, can be simplified to a
+ *   if(a == b){}; // OK
+ *   if(a == false){}; // violation, can be simplified to !a
+ *   if(!(a != true)){}; // violation, can be simplified to a
+ *
+ *   e = (a || b) ? c : d; // OK
+ *   e = (a || false) ? c : d; // violation, can be simplified to a
+ *   e = (a &amp;&amp; false) ? c : d; // violation, can be simplified to false
+ *   e = (a &amp;&amp; b) ? c : d; // OK
+ *
+ * }
```

Comment:
indentation

Reply:
Can you provide more details please?
should I use more spaces for the indentation?
I know I can't use `tab` cause otherwise some checks will fail.

Reply:
This is a closing right curly for method `bar`, it should be indented with either 2 or 4 spaces + 1 space for javadoc.
` *   }` - 2 spaces indent
` * }` - no indent
Method signature for `bar` is not indented correct, by the way

Reply:
It looks like github showing it like it has no indentation. Please correct other items and update screenshot, it will be clearer

Reply:
@wilcoln Please correct indentation.
There should be either 2 or 4 spaces indentation.